packages/react-vapor/src/components/listBox/ListBox.tsx

```diff
@@ -107,9 +108,12 @@ export class ListBox extends React.Component<IListBoxProps, {}> {
 
     render() {
         return (
-            <ul className={this.getClasses()} id={this.props.id}>
-                {this.props.wrapItems(this.getItems())}
-            </ul>
+            <>
+                <ul className={this.getClasses()} id={this.props.id}>
+                    {this.props.wrapItems(this.getItems())}
+                </ul>
+                {this.props.footer}

```

Comment:
This could simply be `this.props.children`, that way you wouldn't need to add a new prop.
Also, it adds more of a "react" feel to it.
```tsx
<ListBox 
    {...props} 
    footer={<span>I am the footer</span>}
/>

// The markup is a little bit more readable (IMO) this way:
<ListBox {...props}>
    <span>I am the footer</span>
</ListBox>
```